xapian-core/include/xapian/termgenerator.h

```diff
@@ -141,6 +144,17 @@ class XAPIAN_VISIBILITY_DEFAULT TermGenerator {
      */
     void set_stemming_strategy(stem_strategy strategy);
 
+    /** Set the stopper strategy.
+     *
+     *  @param strategy The strategy to use - possible values are:
+     *   - STOP_NONE:   Disable stopper for both the stemmed and unstemmed form 
+     *           of the word (Don't use the stopper).  
+     *   - STOP_ALL:    Don't disable stopper for both forms of the word. 
```

Comment:
This would probably be clearer as "Enable stopped for both forms of the word". Generally a positive statement is easier for people to understand than a negative one.